app/models/bank_statement_item.rb

```diff
@@ -53,6 +53,8 @@ class BankStatementItem < Ekylibre::Record::Base
   validates :initiated_on, timeliness: { on_or_after: -> { Time.new(1, 1, 1).in_time_zone }, on_or_before: -> { Time.zone.today + 50.years }, type: :date }, allow_blank: true
   validates :letter, :memo, :transaction_number, length: { maximum: 500 }, allow_blank: true
   validates :name, presence: true, length: { maximum: 500 }
+  validates :started_on, presence: true
```

Comment:
Do not add validations between `[VALIDATORS[ .... ]VALIDATORS]` (they are auto-generated). Add them before or after.